persistent-qq/test/Spec.hs

```diff
@@ -0,0 +1,153 @@
+{-# LANGUAGE QuasiQuotes, GADTs, OverloadedStrings #-}
```

Comment:
This test suite is annoying. We can't have a circular dependency, so in order to depend on `persistent-sqlite` to have a driver to run these tests against, we have to *not* depend on `persistent-test`